components/script/webdriver_handlers.rs

```diff
@@ -356,6 +356,17 @@ pub fn handle_add_cookie(
         .unwrap();
 }
 
+pub fn handle_delete_cookies(documents: &Documents,reply: IpcSender<Result<(),()>>) {
+    for (id, document) in documents.iter() {
```

Comment:
We will want to:
1. choose a particular document
2. get all of its cookies
3. send a message deleting those cookies for this document's URL

Reply:
@jdm is the algorithm compliant now?